xeno/shields.md

```diff
@@ -0,0 +1,112 @@
+## Abstract
+
+<!-- An abstract is a short blurb, about a paragraph or two, succinctly describing your feature. This should mostly be "why", but can include "what". -->
+
+A wider variety of "buff abilities" is used instead of shields. These are applied to limited castes, mainly drone evos and prae.
+
+Buff abilities are explained further below. Shields are kept for castes that only give them to themselves.
+
+Shields are integrated, in some cases, into these buff abilities.
+
+## Goals
+
+1. To replace shields with buff abilities
+
+2. To make said buff abilities intuitive and well telegraphed
+
+3. To make said buff abilities increase survivability and durability but not persist off the front line / long after they are applied.
```

Comment:
This is a good change which i highly agree with.

Reply:
zased